lib/acme/client.rb

```diff
@@ -123,7 +136,7 @@ def kid
     @kid ||= account.kid
   end
 
-  def new_order(identifiers:, not_before: nil, not_after: nil)
+  def new_order(identifiers:, not_before: nil, not_after: nil, eab: nil)
```

Comment:
Why add `eab` here? I'm not sure I follow.

Reply:
No idea actually. I included it based on #210 but ended up deleting it after looking at [7.3.4 in spec](https://www.rfc-editor.org/rfc/rfc8555.html#section-7.3.4) I don't see any reference to it. I forgot to clean it up the argument.